collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java

```diff
@@ -112,23 +111,16 @@ public void collect(CollectRep.MetricsData.Builder builder, long monitorId, Stri
             builder.setMsg("unsupported mongodb diagnostic command: " + command);
             return;
         }
-        ClientSession clientSession = null;
         MongoClient mongoClient;
         CacheIdentifier identifier = null;
         try {
             identifier = getIdentifier(metrics.getMongodb());
             mongoClient = getClient(metrics, identifier);
             MongoDatabase mongoDatabase = mongoClient.getDatabase(metrics.getMongodb().getDatabase());
-            clientSession = mongoClient.startSession();
             CollectRep.ValueRow.Builder valueRowBuilder = CollectRep.ValueRow.newBuilder();
-            Document document;
-            if (metricsParts.length == 1) {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
```

Comment:
hi, if do not use clientSession close, is the mongoDatabase will auto close the seesion after run command?

Reply:
mongoDatabase is used to encapsulate the functions for users to operate the database, and construct the real execution options and hand it over to mongoClient for execution, so it does not need to close.